nixos/modules/services/backup/znapzend.nix

```diff
@@ -411,7 +412,7 @@ in
   };
 
   config = mkIf cfg.enable {
-    environment.systemPackages = [ pkgs.znapzend ];
+    environment.systemPackages = [ pkgs.znapzend ] ++ optional ((filterAttrs (_: v: v.enable && v.mbuffer.enable) cfg.zetup) != {}) pkgs.mbuffer;
```

Comment:
Oh and this shouldn't be needed, there's `mbuffer` in the services `path` below already

Reply:
Yes, it also works without it.